doc/user/assorted_guide.rst

```diff
@@ -1,5 +1,82 @@
+
+.. image:: images/looseCouplingIllustration.png
+    :width: 800
+    :alt: Loose coupling illustration.
+
+Loose coupling is enabled by default in ARMI simulations.
+
+Tight Coupling
+-----------------
+Tight coupling is interpreted as two-way communication between physics within a given time node. Revisiting our previous example, enabling tight coupling results in the temperature distribution being used to generate updated cross sections (new temperatures induce changes such as Doppler broadening feedback) and ultimately an updated power distribution. This process is repeated iteratively until a numerical convergence criteria is met.
+
+.. image:: images/tightCouplingIllustration.png
+    :width: 800
+    :alt: Tight coupling illustration.
+
+The following settings are involved with enabling tight coupling in ARMI:
+
+1. ``tightCoupling``: When ``True``, tight coupling is enabled.
+2. ``tightCouplingSettings``: Used to specify which parameters and convergence criteria will be used to measure the convergence of a given interface.
+
+.. code-block:: yaml
+
+       tightCoupling: true
+       tightCouplingSettings:
+         globalFlux:
+           parameter: power
+           convergence: 1.0e-4
+         thermalHydraulics:
+           parameter: THaverageCladTemperature
+           convergence: 1.0e-2
+
+
+The ``tightCouplingSettings`` settings interact with the interfaces available in ARMI (or an ARMI app) and parameters can be a registered parameter. The ``convergence`` setting within ``tightCouplingSettings`` is expected to be any float value. In the current implementation, different interfaces may have different developer intended restrictions. For example, the global flux solver currently only allows the eigenvalue (i.e. :math:`k_{\text{eff}}`) or block-wise power to be valid ``parameter`` values.
+
+.. warning::
+    The inherent limitations of the above interface-based tight coupling settings have been documented and a new and improved interface is currently being developed.
+
+In the global flux interface, the following norms are used to compute the convergence of :math:`k_{\text{eff}}` and block-wise power.
+
+Eigenvalue
+^^^^^^^^^^
+The convergence of the eigenvalue is measured through an L2-norm.
+
+.. math::
+    \epsilon = \| k_\text{eff} \|_2 = \left( \left( k_\text{eff,old} - k_\text{eff,new} \right)^2 \right) ^ \frac{1}{2}
+
+Block-wise Power
+^^^^^^^^^^^^^^^^
+The block-wise power can be used as a convergence mechanism to avoid the integral effects of :math:`k_{\text{eff}}` (i.e., over and under predictions cancelling each other out) and in turn, can have a different convergence rate. To measure the convergence of the power distribution with the prescribed tolerances (e.g., 1e-4), the power is scaled in the following manner (otherwise the calculation struggles to converge). 
+
+For assembly, :math:`a`, we compute the total power of the assembly,
```

Comment:
minor grammar:  "For an assembly"

Reply:
https://github.com/terrapower/armi/pull/1033/commits/024e5f9f87359fdf98ff53c4a429254cfc40252a